spec/jekyll-feed_spec.rb

```diff
@@ -267,12 +282,4 @@
       expect(contents).to include('<?xml-stylesheet type="text/xml" href="http://example.org/feed.xslt.xml"?>')
     end
   end
-
-  context "with site.lang set" do
-    let(:overrides) { { "lang" => "en-US" } }
-
-    it "should set the language" do
-      expect(contents).to match %r{type="text/html" hreflang="en-US" />}
```

Comment:
How does [this look](https://github.com/jekyll/jekyll-feed/pull/164/files#diff-46663446120a8d6231c2e8f9f752dcfcR160) as a replacement?